nautobot/dcim/views.py

```diff
@@ -2393,56 +2392,7 @@ class CableBulkDeleteView(generic.BulkDeleteView):
 
 
 class ConnectionsListView(generic.ObjectListView):
```

Comment:
The subclasses of this view had bespoke export implementations (below) - need to figure out whether we have equivalent REST endpoints that would provide equivalent data, or possibly add them. These also probably need to be reimplemented for the new UI in any case (out of scope for this PR).

Reply:
Should be doable through the `/api/dcim/cables/` endpoint though it currently lacks a filter for "termination A type OR termination B type matches ...". Might have to add that.